tests/unit/test_host_helpers.py

```diff
@@ -42,7 +42,8 @@ def test_get_interface_with_addr_not_exists(self):
     def test_get_interface_with_addr_exists(self):
         expected = {'br-ens3': {'addresses': ['10.0.0.49'],
                                 'hwaddr': '52:54:00:e2:28:a3',
-                                'state': 'UP'}}
+                                'state': 'UP',
+                                'speed': 'unknown'}}
```

Comment:
Maybe it's worth having one test with an actual value eg: `speed: 10000Mb/s`

Reply:
ack, ill add that